tests/unit/CardsSectionUtilsTest.ts

```diff
@@ -0,0 +1,36 @@
+import getNextBillingDate from '@src/pages/settings/Subscription/CardSection/utils';
+
+describe('getNextBillingDate', () => {
+    beforeAll(() => {
+        jest.useFakeTimers();
+        jest.setSystemTime(new Date(2024, 6, 5));
```

Comment:
NAB, maybe we can add a comment that month is zero-indexed? It tripped me up for a sec 😄

Reply:
Done!